### PR TITLE
Release v6.0.2 lacework-agent Helm Charts

### PR DESCRIPTION
**Pull request by nschmeller**
Signed-off-by: nschmeller <nick.schmeller@lacework.net>

### CODE REVIEW

**Review by cirego — CHANGES_REQUESTED** [review]
Nick, it looks like the pod label change is missing. I would have expected that this PR mostly updated the image tag / chart version and nothing else.

File: lacework-agent/templates/daemonset.yaml [hunk 1]
```diff
@@ -26,9 +26,6 @@ spec:
       {{- end }}
       labels:
         name: {{ include "lacework-agent.name" . }}
```
Comment (joebowbeer) [inline lacework-agent/templates/daemonset.yaml:28]:
```suggestion
        name: {{ include "lacework-agent.name" . }}
        {{- with .Values.laceworkConfig.labels }}
          {{- toYaml . | nindent 8 }}
        {{- end }}
```

File: lacework-agent/templates/daemonset.yaml [hunk 2]
```diff
@@ -26,9 +26,6 @@ spec:
...
-        {{- with .Values.laceworkConfig.labels }}
```
Comment (joebowbeer) [inline lacework-agent/templates/daemonset.yaml:29]: regression

File: lacework-agent/values.schema.json [hunk 3]
```diff
@@ -70,7 +70,10 @@
             "type": "object",
             "required": [
                 "image",
-                "enable"
+                "enable",
+                "clusterType",
+                "scrapeInitialDelayMins",
+                "scrapeIntervalMins"
```
Comment (joebowbeer) [inline lacework-agent/values.schema.json:76]:
```suggestion
                "enable"
```

regression

**Comment by joebowbeer** [comment]
should be no changes to daemonset or schema

**Review by cirego — APPROVED** [review]
Nice! Thank you Nick!

**Pull request merged**